### PR TITLE
Adds new integration [9rpp/securifi]

### PR DESCRIPTION
**Pull request by 9rpp**
Add Securifi RESTful API integration as one of HACS's default repositories

### CODE REVIEW

**Comment by KTibow** [comment]
That repo doesn't exist...

File: integration [hunk 1]
```diff
@@ -87,6 +87,7 @@
   "custom-components/pyscript",
   "custom-components/qbo",
   "custom-components/readme",
+  "custom-components/securifi",
```
Comment (KTibow) [inline integration:90]:
```suggestion
  "9rpp/securifi",
```
And then put it at the top, not here

Comment (9rpp) [reply integration]: Change made as per suggestion

**Comment by 9rpp** [comment]
Pending merges for icons and wheels-custom-integration

**Comment by 9rpp** [comment]
i am not sure what this error is pointing out.  i cant see anything wrong with it. please advise.

Error:  ::error:: hacs.json file is not valid (Expecting ',' delimiter: line 3 column 3 (char 37)).

**Comment by KTibow** [comment]
@9rpp go to https://jsonlint.com/ and validate your `hacs.json` file.

**Comment by 9rpp** [comment]
I've validated my hacs.json on https://jsonlint.com/ and it checks out OK.  Please see the attached screenshot and advise what I can do.

Screenshot from 2020-10-13 14-18-28

**Comment by ludeeus** [comment]
<https://github.com/9rpp/securifi/blob/v1.0/hacs.json> is your released version

**Comment by 9rpp** [comment]
I've created release v1.1 with the hacs.json fixed.  What can I do to update the check to use it?

**Pull request merged**